core/common/src/ContextRealityModel.ts

```diff
@@ -37,7 +37,7 @@ export enum RealityDataProvider {
    * This is the legacy mode where the access to the 3d tiles is harcoded in ContextRealityModelProps.OrbitGtBlob property.
    * It was use to support OrbitPointCloud (OPC) from other server than ContextShare
    * You should use other mode when possible
-   * @see [[RealityDataSource.createRealityDataSourceKeyFromUrl]] that will try to detect provider from an URL
+   * @see [[RealityDataSource.createKeyFromOrbitGtBlobProps]] that will try to detect provider from an URL
```

Comment:
`createKeyFromOrbitGtBlobProps` should be marked `alpha` because `OrbitGtBlobProps` is alpha. (Unfortunately, extract-api will not catch these kinds of mismatches across packages).

Reply:
Good catch! I fix it.